python/tskit/trees.py

```diff
@@ -5552,10 +5567,15 @@ def to_macs(self):
         m = self.get_sequence_length()
         output = [f"COMMAND:\tnot_macs {n} {m}"]
         output.append("SEED:\tASEED")
-        for variant in self.variants(as_bytes=True):
+        for variant in self.variants(return_variant_copies=False):
+            if any(len(allele) > 1 for allele in variant.alleles):
```

Comment:
We should cover this I guess (even if macs output is almost certainly never used now)

Reply:
https://github.com/tskit-dev/tskit/pull/2172/commits/277631d8fef625b19b711d9872707edc8bb25f66